### PR TITLE
Feature/iteration 3 marceline

### PR DESCRIPTION
**Pull request by MarcelineBall**
Requesting code review for functionality of iteration 3, in regards to the new functions and refactoring of earlier functions and event listeners

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -127,7 +128,9 @@ var savedPosters = [];
 var currentPoster;
 
 // event listeners go here 👇
```
Comment (Meekb) [inline src/main.js:130]: WHOA! Great job on figuring this one out team!

File: src/main.js [hunk 2]
```diff
@@ -158,9 +163,20 @@ function getRandomIndex(array) {
 };
 
 function makeRandomPoster() {
```
Comment (Meekb) [inline src/main.js:165]: Looks great -  -  - this is going to be super helpful for refactoring Saturday.

**Review by Meekb — APPROVED** [review]
Great job!

**Review by michannstoner — APPROVED** [review]
Hey team! Great work on this iteration. I think we moved things around and updated functions for the better. It's working and it feels great! Nice work being the driver Marceline, I know this iteration was a lot!

**Comment by Meekb** [comment]
Merging I-3 code.

**Pull request merged**